apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -324,8 +324,15 @@ export default function Success(props: PageProps) {
       return t(`needs_to_be_confirmed_or_rejected${titleSuffix}`);
     }
     if (bookingInfo.user) {
+      const isAttendees = !!bookingInfo.attendees.find((attendee) => attendee.email === session?.user?.email);
+      const attendee = bookingInfo.attendees[0].name || bookingInfo.attendees[0].email;
+      const isHost = bookingInfo.user.email === session?.user?.email;
+
+      const roleBasedAttendees = (isAttendees || isHost) ? "You are" : attendee
+      
```

Comment:
This is still incomplete😓. Will fix it soon

Reply:
@anikdhabal  which scenario ? I can fix if you let me know ?